src/main/java/uk/gov/hmcts/reform/em/hrs/service/ccd/CcdUploadServiceImpl.java

```diff
@@ -72,10 +78,23 @@ private void updateCase(final HearingRecording recording, final HearingRecording
             return;
         }
 
+        if (recordingDto.getRecordingSource() == HearingSource.VH
+            && ccdCaseId != null
+            && recording.getSegments().stream()
+            .anyMatch(x -> recordingDto.getSourceBlobUrl().equals(x.getIngestionFileSourceUri()))
```

Comment:
Could we please avoid using characters like X and use some meaningful short var name?

Reply:
changed